docs/deployment-examples/database-deployments/sql-server/permissions.md

```diff
@@ -0,0 +1,86 @@
+
+- Server Permissions:
+    - `dbcreator`: The ability to create new databases.
+    - `securityadmin`: The ability to create new users and grant them permissions (you will need a check-in place to ensure it doesn't grant random people sysadmin roles).
+- Database Permissions:
+    - `db_ddladmin`: Can run any Data Definition Language (DDL) command in a database.
+    - `db_datareader`: Can read all the data from all user tables.
+    - `db_datawriter`: Can add, delete, or change data from all user tables.
+    - `db_backupoperator`: Can backup the database.
+    - `db_securityadmin`: Can modify role membership and manage permissions.
+    - `db_accessadmin`: Can add or remove access to the database for logins.
+    - Can View Any Definition.
+
+Be sure to assign the deployment account those database roles in the model database.  That is the system database used by SQL Server as a base when a new database is created.  This means the deployment account will be assigned to those roles going forward.
+
+## Fully automated database deployments permission recommendation {#SQLServerdatabases-ManualUsers}
+
+Security admins should be treated the same as system admins, as they can grant permissions at the server level.  For security purposes, it is common to see that role restricted.  In that case, below are the recommended permissions.  It can do everything except create a new SQL Login.
+
+- Server Permissions:
+    - `dbcreator`: The ability to create new databases.
+- Database Permissions:
+    - `db_ddladmin`: Can run any Data Definition Language (DDL) command in a database.
+    - `db_datareader`: Can read all the data from all user tables.
+    - `db_datawriter`: Can add, delete, or change data from all user tables.
+    - `db_backupoperator`: Can backup the database.
+    - `db_securityadmin`: Can modify role membership and manage permissions.
+    - `db_accessadmin`: Can add or remove access to the database for logins.
+    - Can View Any Definition.
+
+## No database creation or user creation, everything else automated permission recommendation
+
+If granting that level of access is not workable or allowed, we recommend the following.  It requires SQL Users to be manually created and the database to already exist.  The process can add existing users to databases as well as deploy everything.
+
+- Database Permissions:
+    - `db_ddladmin`: Can run any Data Definition Language (DDL) command in a database.
+    - `db_datareader`: Can read all the data from all user tables.
+    - `db_datawriter`: Can add, delete, or change data from all user tables.
+    - `db_backupoperator`: Can backup the database.
+    - `db_securityadmin`: Can modify role membership and manage permissions.
+    - `db_accessadmin`: Can add or remove access to the database for logins.
+    - Can View Any Definition.
```

Comment:
@OctopusBob Is a permission missing on this line?

Reply:
View any definition is not a permission, it is a `Grant`.  

```
USE master 
GO 
GRANT VIEW ANY DEFINITION TO User1
```

Reply:
Ah, okay. I'll format those like this:
> Grant `View any definition`.